.github/workflows/main.yml

```diff
@@ -24,7 +24,6 @@ jobs:
     strategy:
       matrix:
         go_version:
-          - "1.17"
```

Comment:
Don't change this. This is out of scope.

Reply:
I changed the minimum version to be >=1.18 in discussion with Boyan Soubachov in this PR: https://github.com/stretchr/testify/pull/1379#discussion_r1188456518. Maybe that was a bad change?